mmv1/third_party/terraform/acctest/gcp_sweeper.go

```diff
@@ -53,3 +55,11 @@ func IsSweepableTestResource(resourceName string) bool {
 	}
 	return false
 }
+
+func AddTestSweepers(name string, sweeper func(region string) error) {
+	_, filename, _, _ := runtime.Caller(0)
+	resource.AddTestSweepers(name+filename, &resource.Sweeper{
+		Name: name,
+		F:    sweeper,
+	})
```

Comment:
```suggestion
	_, filename, _, _ := runtime.Caller(0)
	hash := crc32.NewIEEE()
	hash.Write([]byte(filename))
	hashedFilename := hex.EncodeToString(hash.Sum(nil))
	uniqueName := name + "_" + hashedFilename
	
	resource.AddTestSweepers(uniqueName, &resource.Sweeper{
		Name: name,
		F:    sweeper,
	})
```

suggestion to save memory on the users machine by hashing the filename. What do you think? Otherwise we would be storing a lot of redundant data in the ram of the users machine.

new dependencies
```
	"encoding/hex"
	"hash/crc32"
```

Reply:
Yeah, that is a great point.